generatebundlefile/hack/release_staging.sh

```diff
@@ -84,8 +84,12 @@ function push () {
     local version=${1?:no version specified}
     cd "${BASE_DIRECTORY}/generatebundlefile/output-${version}"
     awsAuth "$REPO" | "$ORAS_BIN" login "$REPO" --username AWS --password-stdin
-    "$ORAS_BIN" push "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
-    "$ORAS_BIN" push "${REPO}:v${version}-latest" bundle.yaml
+    "$ORAS_BIN" pull "${REPO}:v${version}-latest" -o ${version}
+    if (git diff --no-index --quiet -- ${version}/bundle.yaml bundle.yaml) then
```

Comment:
it looks like this is just comparing two files. Wouldn't `diff -q` work here?

Reply:
This is the command I tested with in dev. I believe `--no-index` allows it to compare two files or directories, but if you don't include that it will only accept 1 file and compare to the git index.

```diff only accepts a single pair of files, or a directory and a list of files.```